### PR TITLE
Mks 23 create web api project

### PR DESCRIPTION
**Pull request by TsisinskyjV**

### CODE REVIEW

**Review by tann96 — APPROVED** [review]

File: Volodymyr_Tsisinskyi/.gitignore.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Nedo_net/.vs/
```
Comment (bvzayats) [inline Volodymyr_Tsisinskyi/.gitignore.txt]: Why in the PR included files from vs, obj folders? Revert this files in next commit.

Comment (TsisinskyjV) [reply Volodymyr_Tsisinskyi/.gitignore.txt]: done

**Review by Magfiozy — APPROVED** [review]

File: Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/ValuesController.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
+﻿using System;
```
Comment (bvzayats) [inline Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/ValuesController.cs]: Remove unused namespaces.

Comment (TsisinskyjV) [reply Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/ValuesController.cs]: done

**Review by bvzayats — APPROVED** [review]

**Review by bvzayats — COMMENTED** [review]
Update this branch with development!

**Pull request merged**